daffodil-test/src/test/scala/org/apache/daffodil/section05/facets/TestPatternRanges.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.daffodil.section05.facets
+
+import org.apache.daffodil.tdml.Runner
+import org.junit.AfterClass
+import org.junit.Test
+
+object TestPatternRanges {
+  lazy val runner = Runner("/org/apache/daffodil/section05/facets", "PatternRanges.tdml")
+
+  @AfterClass def shutDown = {
+    runner.reset
+  }
+}
+
+import org.apache.daffodil.section05.facets.TestPatternRanges._
+
+class TestPatternRanges {
+
```

Comment:
Most TDML tests would put a relative import at this line instead of a absolute-package import at line 31.

Reply:
I'll hoist this in.